### ISSUE TITLE
Chapter 4 user.png

### ISSUE THREAD

**Issue by Lethu360** [dabit3/react-native-in-action#6]
i'm struggling to find user.png used in Listing 4.10 where can i find chapter 4.

**Comment by eprikazc** [comment]
Indeed, looks like `user.png` is missing. @dabit3 - is it possible to add it?

**Comment by bartonstanley** [comment]
I also do not see user.png.
